include/rtl_433_devices.h

```diff
@@ -231,6 +231,7 @@
     DECL(fineoffset_wn34) \
     DECL(rubicson_pool_48942) \
     DECL(badger_orion) \
+    DECL(govee_h5054)
```

Comment:
needs a `\` at the end

Reply:
fyi - it seems like clang-format strips it out, but i added it back in manually